File: javascript/7_kyu/squares_sequence.js
// Squares Sequence
// https://www.codewars.com/kata/5546180ca783b6d2d5000062/train/javascript

function squares(x, n) {
    var ans = [];

    for (var i = 0; i < n; i++) {
        ans.push(x);
        var x = x**2;
    }

    return ans;
}

console.log(squares(2, 5))

File: python/7_kyu/squares_sequence.py
# Squares Sequence
# https://www.codewars.com/kata/5546180ca783b6d2d5000062/train/python

def squares(x, n):
    ans = []

    while len(ans) < n:
        ans.append(x)
        x = x**2
    
    return ans



print(squares(2, 5))